inference-engine/tools/compile_tool/main.cpp

```diff
@@ -184,6 +193,20 @@ static std::map<std::string, std::string> parsePrecisions(const std::string &iop
 }
 
 using supported_precisions_t = std::unordered_map<std::string, InferenceEngine::Precision>;
+using supported_layouts_t = std::unordered_map<std::string, InferenceEngine::Layout>;
+using matchLayoutToDims_t = std::unordered_map<InferenceEngine::Layout, size_t>;
```

Comment:
It looks like maxOS STL doesn't like enums as `std::unordered_map` keys. Please add explicit `std::hash` specialization for `InferenceEngine::Layout` to fix MacOS build.

Reply:
Fixed.